src/App.tsx

```diff
@@ -1,9 +1,21 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
 
 export const App: React.FC = () => {
+  const [query, setQuery] = useState('');
+  const preparedChangedQuery = query.toLowerCase();
```

Comment:
```suggestion
  const preparedChangedQuery = query.toLowerCase().trim();
```
<img width="507" alt="image" src="https://user-images.githubusercontent.com/55802630/184820164-6d688925-cb46-4bff-b9b5-afa144837396.png">